src/constants.js

```diff
@@ -3,5 +3,6 @@ export const GREEN = [0, 255, 0, 230];
 export const GREY = [0, 0, 0, 230];
 export const BASE_URL = 'https://portal.rainforestqa.com';
 export const WORK_AVAILABLE_URL = `${BASE_URL}/api/1/testers/`;
+export const ABANDONED_URL = 'https://tester.rainforestqa.com/abandoned';
```

Comment:
It's the URL Rainforest sends you to when you abandon a job

On Aug 10, 2016 02:08, "Russell Smith" <notifications@github.com> wrote:

> In src/constants.js
> <https://github.com/rainforestapp/tester-chrome-extension/pull/238#discussion_r74100854>
> :
>
> > @@ -3,5 +3,6 @@ export const GREEN = [0, 255, 0, 230];
> >  export const GREY = [0, 0, 0, 230];
> >  export const BASE_URL = 'https://portal.rainforestqa.com';
> >  export const WORK_AVAILABLE_URL = `${BASE_URL}/api/1/testers/`;
> > +export const ABANDONED_URL = 'https://tester.rainforestqa.com/abandoned';
>
> @helloworld111gh <https://github.com/helloworld111gh> where did this come
> from?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rainforestapp/tester-chrome-extension/pull/238/files/8d85175ae8dce4177ae7ec12d48500b1edbbc1bb#r74100854>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJs-6XM0BJkXMPds9lPQ5WwPHa9s3lf1ks5qeLP4gaJpZM4I6PlG>
> .
>